stabilizer/src/main/java/com/hazelcast/stabilizer/worker/tasks/AbstractWorkerTask.java

```diff
@@ -37,7 +38,7 @@
     long iteration = 0;
 
     public AbstractWorkerTask() {
-        this.selector = createOperationSelector();
+        this.selector = createOperationSelectorBuilder().build();
```

Comment:
I don't think this belongs in the abstract worker. I want to prevent adding too much framework and certainly want to prevent forcing a user to implement methods if he doesn't care about them.

Afaik the OperationSelector can also be initialized when the worker is created in the concrete subclass

```
class FooWorker extends AbstractWorker{
     OperationSelector<FoOOPeration> selector = selectorBuilder.newInstance();
     ....
}
```